src/Parser/Rogue/Subtlety/Modules/RogueCore/SymbolsDamageTracker.js

```diff
@@ -0,0 +1,16 @@
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import FilteredDamageTracker from '../CastTracker/FilteredDamageTracker';
+
+class SymbolsDamageTracker extends FilteredDamageTracker {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  
+  shouldProcessEvent(event) {
+    return this.combatants.selected.hasBuff(SPELLS.SYMBOLS_OF_DEATH.id);
+  }
+}
+
```

Comment:
The two damage tracker are fine as is, but you could probably merge them and have two shouldProcessEvent calls called something like shouldProcessEventSymbols and shouldProcessEventDance or whatever, not a needed change, but maybe something to consider.

Reply:
I defiantly want them separate, because I'm using the base DamageTracker functionality to aggregate all data. Having 1 tracker with multiple aggregates would be similar in usage, but harder to maintain in my opinion.